wagtailio/settings/production.py

```diff
@@ -40,6 +21,8 @@
 except ValueError:
     VERCEL_DEPLOY_MAX_WORKERS = 10
 
+MANIFEST_LOADER["cache"] = True  # noqa
```

Comment:
Question: Why does this need no QA?

Reply:
It was from https://github.com/wagtail/guide/blob/main/apps/guide/settings/production.py#L5

It's a bit out of my remit but i'm presuming there's no reason not to check this line?

Reply:
It's most likely upset that there's an assignment without an explicit import or variable creation. Leave as is.